.github/workflows/ci-release-build.yml

```diff
@@ -0,0 +1,38 @@
+on: push
+name: Build & Release
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@master
+      with:
+        dotnet-version: '6.0.x'
+        include-prerelease: true
+    - name: Build
+      shell: bash
+      run: |
+        dotnet build NStrip -c Release
+        mkdir -p publish
+        cp -a NStrip/bin/Release/*/NStrip.exe publish/
+    - name: Release Latest
+      uses: marvinpinto/action-automatic-releases@latest
+      if: github.ref == 'refs/heads/master'
+      with:
+        repo_token: "${{ secrets.GITHUB_TOKEN }}"
+        automatic_release_tag: "latest"
+        title: "Latest"
+        prerelease: true
+        files: |
+          publish/NStrip.exe
```

Comment:
Instead of pushing a prerelease on every single push to master and cluttering the releases page, I instead suggest just saving an artefact. In that case, we can then add a URL to [nightly.link](https://nightly.link/) for those who need builds from master.

Moreover, saving artifacts automatically is useful for PRs as we can easily test them without pulling and building manually.

Reply:
It overwrites the same release so there is only one up-to-date release in releases, or what page do you refer to?

Reply:
Fair enough, although I'd still rather it only generate artefacts from builds instead of prereleases. One can then also lift the whole limitation of only saving the build results for master branch.

Reply:
I have added an artifact upload and removed the master auto-release.